### PR TITLE
adding lido.TokenRebased event table

### PR DESCRIPTION
**Pull request by ppclunghe**
## What?
Add, please, lido.TokenRebased event to data schema

## How? 
I used [abi-parser](https://nansen-contract-parser-prod.web.app/) to build the table definitions

### CODE REVIEW

File: dags/resources/stages/parse/table_definitions/lido/Lido_event_TokenRebased.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "sharesMintedAsFees",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "<TABLE_PREFIX>_event_TokenRebased"
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/lido/Lido_event_TokenRebased.json:89]: Can you update the name of the `table_name` to be `Lido_event_TokenRebased`?

File: dags/resources/stages/parse/table_definitions/lido/Lido_event_TokenRebased.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            ],
+            "name": "TokenRebased",
+            "type": "event"
+        },
+        "contract_address": "0xae7ab96520de3a18e5e111b5eaab095312d7fe84",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "<INSERT_DATASET_NAME>",
```
Comment (kome12) [inline dags/resources/stages/parse/table_definitions/lido/Lido_event_TokenRebased.json:50]: Can you update the `dataset_name` to be `lido`?

**Review by kome12 — COMMENTED** [review]
Added some comments to fix the checks!

**Comment by ppclunghe** [comment]
Thanks @kome12
fixed but tests still don't pass

**Comment by kome12** [comment]
> Thanks @kome12 fixed but tests still don't pass

That test is fine to ignore for now. There seems to be some issue with the underlying package.

**Review by kome12 — APPROVED** [review]

**Pull request merged**